src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -160,6 +160,27 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 
 	// MARK: Initial Screens
 
+	private func showOverrideTestNotice(
```

Comment:
Did I miss the call to show the VC somewhere?

Reply:
not used here now - will get used in the next ticket